### PR TITLE
Remove npm minor grouping from renovate config

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

The renovate config in this repository groups patch and minor updates together. I think this is unhelpful as it has led to this enormous PR (which is failing): https://github.com/bitwarden/directory-connector/pull/353.

I'd rather smaller, easier PRs, even if we have many of them. We don't use this grouping in `clients` either.

The proposal here is to remove this grouping (this PR) and then force renovate to create new update PRs for each dependency (not sure how to do this - @withinfocus?).

@withinfocus renovate is your area, do you have any feedback on this approach? 

@bitwarden/team-admin-console-dev this is going to affect the team, so feedback welcome.

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -1,17 +1,12 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>bitwarden/renovate-config"],
-  "enabledManagers": ["github-actions", "npm"],
```
Comment (withinfocus) [inline .github/renovate.json:4]: ❌ This removes NPM updates altogether.

**Review by withinfocus — CHANGES_REQUESTED** [review]
You have it correct -- just remove that group and these will come in individually. I am not sure we do this for NPM anywhere else, but those projects are more-aggressively merged.

File: .github/renovate.json (hunk 1 above)
Comment (eliykat) [reply .github/renovate.json]: Fixed!

**Comment by eliykat** [comment]
@withinfocus to clarify, will renovate automatically detect the updated config, close the existing PR, and open the new PRs?

**Review by withinfocus — APPROVED** [review]

**Comment by withinfocus** [comment]
> @withinfocus to clarify, renovate should automatically detect the updated config, close the existing PR, and open the new PRs?

It'll run again when the config changes, but it might not necessarily make _new_ PRs given the schedule. That's easily worked around there where you just ask for PR creation via the checkboxes at https://github.com/bitwarden/directory-connector/issues/303.

**Review by r-tome — APPROVED** [review]

**Pull request merged**